common/scripted_effects/00_scripted_effects.txt

```diff
@@ -18762,11 +18751,9 @@ china_war_spawn_troops_effect = {
 					home = event_target:spawn_province
 					troops = {
 						archers = { 144 144 }
-						light_infantry = { 203 203 }
-						heavy_infantry = { 66 66 }
-						pikemen = { 19 19 }
```

Comment:
I would summarize heavy and light infantry... or even give mages or something else.